src/lightning/pytorch/strategies/ddp.py

```diff
@@ -419,6 +419,39 @@ def teardown(self) -> None:
         super().teardown()
 
 
+class MultiModelDDPStrategy(DDPStrategy):
+    @override
+    def _setup_model(self, model: Module) -> Module:
```

Comment:
The typing is not very happy here as the parent class has the following footprint:
```py
def _setup_model(self, model: Module) -> DistributedDataParallel:
```